samples/csharp_dotnetcore/11.qnamaker/README.md

```diff
@@ -1,52 +1,52 @@
-
-# To try this sample
-
-- Clone the samples repository
-```bash
-git clone https://github.com/Microsoft/botbuilder-samples.git
-```
-- [Optional] Update the `appsettings.json` file under `botbuilder-samples\samples\csharp_dotnetcore\11.qnamaker` with your botFileSecret.  For Azure Bot Service bots, you can find the botFileSecret under application settings.
-# Prerequisites
-- Follow instructions [here](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/set-up-qnamaker-service-azure)
-to create a QnA Maker service.
-- Follow instructions [here](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/migrate-knowledge-base) to
-import the [sample.qna](sample.qna) to your newly created QnA Maker service.
-- Update [BotConfiguration.bot](BotConfiguration.bot) with your kbid (KnowledgeBase Id) and endpointKey in the "qna" services section. You can find this
-information under "Settings" tab for your QnA Maker Knowledge Base at [QnAMaker.ai](https://www.qnamaker.ai)
-- (Optional) Follow instructions [here](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/QnAMaker) to set up the
-QnA Maker CLI to deploy the model.
-
-## Visual Studio
-- Navigate to the samples folder (`botbuilder-samples\samples\csharp_dotnetcore\11.qnamaker`) and open QnABot.csproj in Visual Studio .
-- Hit F5
-
-## Visual Studio Code
-- Open `botbuilder-samples\samples\csharp_dotnetcore\11.qnamaker` sample folder.
-- Bring up a terminal, navigate to `botbuilder-samples\samples\csharp_dotnetcore\11.qnamaker` folder.
-- Type 'dotnet run'.
-
+﻿This sample shows how to integrate QnA Maker in a simple bot with ASP.Net Core 2. 
+
+# Concepts introduced in this sample
+The [QnA Maker Service](https://www.qnamaker.ai) enables you to build, train and publish a simple question
+and answer bot based on FAQ URLs, structured documents or editorial content in minutes.
+
+In this sample, we demonstrate how to use the QnA Maker service to answer questions based on a FAQ text file as input.
+
+# To try this sample
+
+- Clone the samples repository
+```bash
+git clone https://github.com/Microsoft/botbuilder-samples.git
+```
+- [Optional] Update the `appsettings.json` file under `botbuilder-samples\samples\csharp_dotnetcore\11.qnamaker` with your botFileSecret.  For Azure Bot Service bots, you can find the botFileSecret under application settings.
+# Prerequisites
+- Follow instructions [here](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/set-up-qnamaker-service-azure)
+to create a QnA Maker service.
+- Follow instructions [here](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/tutorials/migrate-knowledge-base) to
+import the [sample.qna](https://github.com/Microsoft/BotBuilder-Samples/blob/v4/samples/csharp_dotnetcore/11.qnamaker/CognitiveModels/sample.qna) to your newly created QnA Maker service.

```

Comment:
This is a bad idea, it should be a relative path to the `sample.qna` file. If the sample is moved or renamed this link will need to be changed again.

Try `CognitiveModels/sample.qna` instead.